test/statmodels.jl

```diff
@@ -18,6 +18,14 @@ x1    1.45666         Good    1  -12.56  0.1200
 x2  -23.14            Great  56    0.13  0.3467
 x3    1.56734e-13     Bad     2    0.00  <1e-15
 ───────────────────────────────────────────────"""
+
+@test sprint(show, MIME"text/markdown"(), ct) == """
+|    |     Estimate  |Comments  |df  |     t  |     p  |
+|:---|--------------:|---------:|---:|-------:|:-------|
```

Comment:
Is `:|:` intended?

Reply:
Markdown tables use which side the `:` is on to denote justification, so `t` has one justification and `p` has the opposite